README.md

```diff
@@ -101,12 +101,12 @@ default_context:
 full_name: "Audrey Roy"
 email: "audreyr@gmail.com"
 github_username: "audreyr"
-cookiecutters_dir: "~/.cookiecutters/"
+cookiecutters_dir: "~/$XDG_CACHE_HOME/cookiecutters/"
 ```
 
-* Cookiecutters (cloned Cookiecutter project templates) are put into ``~/.cookiecutters/`` by default, or cookiecutters_dir if specified.
```

Comment:
Correct it to `$XDG_CACHE_HOME/cookiecutters/` and optionally include link to the XDG spec to sort two issues: what is the default when variable is undefined and also to allow reader to learn avout XDG spec.